--- final-doom-tnt-1.9/doom2-states	2008-07-28 12:49:49.000000000 +0100
+++ chex-quest/chex-states	2008-07-28 12:50:08.000000000 +0100
@@ -535,9 +535,9 @@
 {"misc1":0,"spritenum":42,"misc2":0,"frame":2,"tics":3,"action":30,"nextstate":535}
 {"misc1":0,"spritenum":42,"misc2":0,"frame":3,"tics":3,"action":30,"nextstate":536}
 {"misc1":0,"spritenum":42,"misc2":0,"frame":3,"tics":3,"action":30,"nextstate":529}
-{"misc1":0,"spritenum":42,"misc2":0,"frame":4,"tics":8,"action":31,"nextstate":538}
-{"misc1":0,"spritenum":42,"misc2":0,"frame":5,"tics":8,"action":31,"nextstate":539}
-{"misc1":0,"spritenum":42,"misc2":0,"frame":6,"tics":8,"action":56,"nextstate":529}
+{"misc1":0,"spritenum":42,"misc2":0,"frame":4,"tics":3,"action":31,"nextstate":538}
+{"misc1":0,"spritenum":42,"misc2":0,"frame":5,"tics":3,"action":31,"nextstate":539}
+{"misc1":0,"spritenum":42,"misc2":0,"frame":6,"tics":0,"action":56,"nextstate":529}
 {"misc1":0,"spritenum":42,"misc2":0,"frame":7,"tics":2,"action":0,"nextstate":541}
 {"misc1":0,"spritenum":42,"misc2":0,"frame":7,"tics":2,"action":25,"nextstate":529}
 {"misc1":0,"spritenum":42,"misc2":0,"frame":8,"tics":8,"action":0,"nextstate":543}
